composer/loggers/mosaicml_logger.py

```diff
@@ -44,7 +50,25 @@ class MosaicMLLogger(LoggerDestination):
     """Log to the MosaicML platform.
 
     Logs metrics to the MosaicML platform. Logging only happens on rank 0 every ``log_interval``
-    seconds to avoid performance issues.
+    seconds to avoid performance issues. The following metrics are logged upon ``INIT``:
+    - ``composer/autoresume``: Whether or not the run can be stopped / resumed during training.
+    - ``composer/precision``: The precision to use for training.
+    - ``composer/train_loader_workers``: The number of workers for the train dataloader.
+    - ``composer/eval_loaders``: A list of dictionaries containing the label and the number of workers for each
+    evaluation dataloader.
+    - ``composer/optimizers``: A list of dictionaries containing the _ for each opimizer.
+    - ``composer/algorithms``: A list of dictionaries containing the _ for each algorithm.
```

Comment:
what is `_`?

Reply:
oops -- placeholder a left in there. just changed this to `A list of dictionaries containing information for each <optimizer / algorithm>`